src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -528,7 +540,7 @@ private string GetInOutPortsData(NodeModel node)
             stringBuilder.Append(inPorts.Any() ? inPorts.Aggregate((i, j) => i + "," + j) : "");
             stringBuilder.Append("], \"OutPorts\": [");
             stringBuilder.Append(outPorts.Any() ? outPorts.Aggregate((i, j) => i + "," + j) : "");
-            stringBuilder.Append("], \"Data\": \"" + GetValue(node) + "\"}");
+            stringBuilder.Append("]}");
```

Comment:
I failed to see where do we retrieve the data in `GetInOutPortsData` method. Just want to make sure this is intended before I merge this in :)